features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/RumConfiguration.kt

```diff
@@ -283,6 +284,20 @@ data class RumConfiguration internal constructor(
             return this
         }
 
+        /**
+         * Sets the configuration for the slow frames listener, which is used to compute the slow frames array,
+         * slow frames ratio, and freeze ratio.
+         *
+         * Setting this property to null will disable the slow frames listener and stop the corresponding ratio computation.
```

Comment:
We also need to mention what is the default configuration value (e.g. slow frames collection is disabled or enabled by default).

Reply:
I plan to add this after dogfood when we would ready to deploy this feature to production